### PR TITLE
Removed terratest because terraform bin is now 0.14.11

### PR DESCRIPTION
**Pull request by soutenniza**
## Changes proposed in this pull request:
- Terratest is removed so default "terraform" is used.

## security considerations
Nope

### CODE REVIEW

File: ci/pipeline.yml [hunk 1]
```diff
@@ -44,7 +44,6 @@ jobs:
       inputs:
       - name: pull-request
       params:
-        TERRAFORM_BIN: terratest-14.11
```
Comment (ccostino) [inline ci/pipeline.yml:47]: Does this `params` section need to stay now that there's nothing in it?

**Review by ccostino — COMMENTED** [review]
Thanks, @soutenniza!  One quick question.

File: ci/pipeline.yml (hunk 1 above)
Comment (soutenniza) [reply ci/pipeline.yml:47]: Better to leave it there in case we need it again

**Review by ccostino — APPROVED** [review]
Thanks again, @soutenniza!

**Pull request merged**